### PR TITLE
Update project to be compatible with Quorum 0.8.0

### PR DESCRIPTION
**Pull request by JohnChangUK**
Updating the project to be compatible with Quorum 0.8.0.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,6 +3,7 @@ plugins {
 }
 
 apply plugin: 'java'
+'idea'
```
Comment (conor10) [inline build.gradle]: You'll need `apply plugin: 'idea'` here

File: src/main/java/io/blk/erc20/generated/HumanStandardToken.java [hunk 2]
```diff
@@ -117,83 +119,83 @@ public ApprovalEventResponse call(Log log) {
         });
     }
```
Comment (conor10) [inline src/main/java/io/blk/erc20/generated/HumanStandardToken.java]: Did you regenerate this file using the command line tools? As it was a generated file it shouldn't be manually updated.

File: build.gradle [hunk 3]
```diff
@@ -18,7 +19,8 @@ dependencies {
             'io.springfox:springfox-swagger2:2.7.0',
             'io.springfox:springfox-swagger-ui:2.7.0',
             'org.projectlombok:lombok:1.16.16',
-            'org.web3j:quorum:0.4.3'
+            'org.web3j:quorum:0.8.0',
+            'org.springframework.boot:spring-boot-configuration-processor'
```
Comment (conor10) [inline build.gradle]: What's the configuration processor dependency for?

File: src/main/java/io/blk/erc20/Controller.java [hunk 4]
```diff
@@ -50,7 +51,7 @@ NodeConfiguration config() {
     @RequestMapping(value = "/deploy", method = RequestMethod.POST)
     String deploy(
             HttpServletRequest request,
-            @RequestBody ContractSpecification contractSpecification) {
+            @RequestBody ContractSpecification contractSpecification) throws Exception {
```
Comment (conor10) [inline src/main/java/io/blk/erc20/Controller.java]: What is your rationale for allowing the Exception to be thrown here?

File: src/main/java/io/blk/erc20/generated/HumanStandardToken.java (hunk 2 above)
Comment (JohnChangUK) [reply src/main/java/io/blk/erc20/generated/HumanStandardToken.java]: Right, so once compiled, I need to adjust the code according to the generated file HumanStandardToken.java, correct?

File: build.gradle (hunk 1 above)
Comment (JohnChangUK) [reply build.gradle]: Got it, thanks

File: src/main/java/io/blk/erc20/Controller.java (hunk 4 above)
Comment (JohnChangUK) [reply src/main/java/io/blk/erc20/Controller.java]: The deploy() method in Contract Service throws a RunTimeException, due to the send() method which comes from RemoteCall class. I saw that get() was replaced with send() in the updated commits in Web3j.

File: build.gradle (hunk 3 above)
Comment (JohnChangUK) [reply build.gradle]: Was not needed, thanks!

File: src/main/java/io/blk/erc20/Controller.java (hunk 4 above)
Comment (conor10) [reply src/main/java/io/blk/erc20/Controller.java]: 👍

File: src/main/java/org/web3j/erc20/generated/HumanStandardToken.java [hunk 5]
```diff
@@ -0,0 +1,248 @@
+package org.web3j.erc20.generated;
```
Comment (conor10) [inline src/main/java/org/web3j/erc20/generated/HumanStandardToken.java]: You'll need to genererate this with the  correct package name - its different to what it was.

File: src/main/java/io/blk/erc20/generated/HumanStandardToken.java (hunk 2 above)
Comment (JohnChangUK) [reply src/main/java/io/blk/erc20/generated/HumanStandardToken.java]: Changed the location of the generated folder to the correct place again, path = 'src/main/java/io/blk/erc20/generated/HumanStandardToken.java'

File: src/main/java/io/blk/erc20/ContractService.java [hunk 6]
```diff
@@ -195,15 +188,15 @@ private HumanStandardToken load(String contractAddress) {
                 contractAddress, quorum, transactionManager, GAS_PRICE, GAS_LIMIT);
     }
 
-    private <T> T extractValue(Type<T> value) {
+    private <T> T extractValue(T value) {
```
Comment (conor10) [inline src/main/java/io/blk/erc20/ContractService.java]: You no longer need this method.

Comment (JohnChangUK) [reply src/main/java/io/blk/erc20/ContractService.java]: I can still see 3 methods using extractValue().  is using the method. extractLongValue(), symbol(), version() are the method.

**Comment by conor10** [comment]
Great stuff.

**Pull request merged**